manifests/eap/client.pp

```diff
@@ -69,21 +69,21 @@
     $ssl_ca_file_path="cacert.pem"
 )
 {
+    include lightblue::eap::client::modulepath
```

Comment:
If this is changed to `require` instead of `include` you don't need the require on line 86 for `file { $module_dirs :`